### DISCUSSION TITLE
why  this error occured in razorpay payment getway

### DISCUSSION THREAD

**Discussion by salmanansari14** [community/community#126715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

RROR in ./node_modules/razorpay/dist/utils/razorpay-utils.js 69:15-32
Module not found: Error: Can't resolve 'crypto' in 'C:\Users\Salman\Desktop\llll\payment-getway\node_modules\razorpay\dist\utils'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
	- install 'crypto-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "crypto": false }
why did this error occurred in mernn project

**Comment by queenofcorgis** [comment]
Hi @salmanansari14 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Razorpay. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
